src/node/rpc/memberfrontend.h

```diff
@@ -765,19 +765,24 @@ namespace ccf
           if (!key_share_info.has_value())
           {
             return make_error(
-              HTTP_STATUS_BAD_REQUEST, "No key share info available");
+              HTTP_STATUS_INTERNAL_SERVER_ERROR, "No key share info available");
           }
           std::vector<uint8_t> decrypted_ls(LedgerSecret::MASTER_KEY_SIZE);
           crypto::GcmCipher encrypted_ls;
           encrypted_ls.deserialise(key_share_info->encrypted_ledger_secret);
```

Comment:
Can the deserialisation also fail?

Reply:
I think so but it will be covered by the generic PC exception catch?

Reply:
BTW, this code will be moved to a new global hook when the shares get stored in the KV.